frontend/app/.server/redis.ts

```diff
@@ -37,7 +37,7 @@ function getRedisConfig(): RedisOptions {
     REDIS_USERNAME,
   } = serverEnvironment;
 
-  const redisPassword = Redacted.value(REDIS_PASSWORD);
+  const redisPassword = REDIS_PASSWORD && Redacted.value(REDIS_PASSWORD);
```

Comment:
Is this `&&` required here?

Reply:
![image](https://github.com/user-attachments/assets/98e0cbe4-5293-4d47-af86-3f4b462fb6f2)